applications/tari_base_node/src/main.rs

```diff
@@ -128,10 +129,12 @@ fn main() {
             return;
         },
     };
-
-    // Configure the shutdown daemon to listen for CTRL-C
     let flag = node.get_flag();
-    handle_ctrl_c(&rt, flag);
+
+    // Configure the cli
+    rt.spawn(async move {
+        cli_loop(flag).await;
+    });
```

Comment:
Should work and will generate less code
```suggestion
   rt.spawn(cli_loop(flag));
```